move-to-sdk/app/io/sphere/sdk/utils/FutureUtils.java

```diff
@@ -29,4 +32,30 @@ private FutureUtils() {
                         .map(CompletableFuture::join)
                         .collect(Collectors.<T>toList()));
     }
+
+    public static <T> CompletionStage<T> exceptionallyCompletedFuture(final Throwable throwable) {
```

Comment:
@schleichardt I saw after I created the method that you had the same in the utils. The name though (although shorter, and I'm pro shorter names) doesn't fit very well the code when reading.
Having into mind that the positive return looks like:
`return completedFuture(value);`
Then the negative return for me is:
`return exceptionallyCompletedFuture(throwable);`

With your method could look like:
`return failed(throwable);`
`return CompletableFutureUtils.failed(throwable);`
So I'm not that inclined to use yours, that's why I left mine.